layers/meta-balena-coral/recipes-core/images/resin-image.bbappend

```diff
@@ -0,0 +1,6 @@
+include resin-image.inc
+
+IMAGE_INSTALL_append = " \
+    imx-board-wlan \
```

Comment:
Does this mean that the flasher image does not have a working wlan?

Reply:
@floion thanks, added it for both images, as well as overriding serial_consoles only for this specific machine